src/lib.rs

```diff
@@ -108,14 +108,24 @@ pub enum ShaderStage {
 #[cfg_attr(feature = "deserialize", derive(Deserialize))]
 #[allow(missing_docs)] // The names are self evident
 pub enum StorageClass {
-    Constant,
+    /// Function locals.
     Function,
+    /// Pipeline output, per invocation.
```

Comment:
This comment, and the next one: "/// Pipeline input, per invocation, mutable.", are swapped. But, ultimately they are very handy to have, as I now instantly know where to use them for.